Draft package vignettes (#17)
CityRiverSpaces/CRiSp: vignettes/corridor-delineation.Rmd

)
```

```{r setup, warning=FALSE, message=FALSE}
library(CRiSp)
library(dplyr)
library(sf)
library(osmdata)
```

> In this notebook we explore how to delineate a river corridor using Bucharest as the study area. We focus on one of the rivers and use a specific projected CRS for the analysis. Also, we make sure that we include a given area around the city boundaries.

Author: ```suggestion
In this notebook we explore how to delineate a river corridor using Bucharest as the study area. We focus on one of the rivers and use a specific projected CRS for the analysis. A project CRS is required for calculating distances along the street network during delineation. Also, we make sure that we include a given area around the city boundaries.
```

```{r variables}
city_name <- "Bucharest, Romania"   # Be specific and spell as in OSM
river_name <- "Dâmbovița"           # Spell as it appears in OpenStreetMap

> epsg_code <- 32635                  # UTM zone 35N

Author: Thanks for pointing this out. I added a sentence to explain why we work with a projected CRS.

> In this notebook we explore how to delineate a river corridor using Bucharest as the study area. We focus on one of the rivers and use a specific projected CRS for the analysis. A project CRS is required for calculating distances along the street network during delineation. Also, we make sure that we include a given area around the city boundaries.

Reviewer: ```suggestion
In this notebook we explore how to delineate a river corridor using Bucharest as the study area. We focus on one of the rivers and use a specific projected CRS (coordinate reference system) for the analysis. A project CRS is required for calculating distances along the street network during delineation. Also, we make sure that we include a given area around the city boundaries.
```

> city_name <- "Bucharest, Romania"   # Be specific and spell as in OSM

Reviewer: ```suggestion
city_name <- "Bucharest, Romania"   # Be specific and spell as in OpenStreetMap (OSM)
```
Given the instruction below it would make sense to be consistent in wording used

> river_name <- "Dâmbovița"           # Spell as it appears in OpenStreetMap

Reviewer: ```suggestion
river_name <- "Dâmbovița"           # Spell as in OpenStreetMap (OSM)
```
